src/api/db/migrate/20170630144825_convert_tokens_to_service_tokens.rb

```diff
@@ -0,0 +1,9 @@
+class ConvertTokensToServiceTokens < ActiveRecord::Migration[5.1]
+  def up
+    Token.where(type: nil).update_all(type: 'Token::Service')
+  end
+
+  def down
+    Token::Service.update_all(type: nil)
```

Comment:
Just to leave the things as they were?... Not super-mandatory, but anyway to be consistent